packages/patternfly-4/react-core/src/components/Page/PageSection.js

```diff
@@ -7,7 +7,9 @@ export const PageSectionVariants = {
   default: 'default',
   light: 'light',
   dark: 'dark',
-  darker: 'darker'
+  darker: 'darker',
+  fill: 'fill',
+  no_fill: 'no_fill',
```

Comment:
I'm not sure if we've defined this anywhere - but the underscore seems unusual in the naming here.  I think we should chat about standardizing on camelCase as we do with other variable names and style names.  @tlabaj - do we have this documented?

Reply:
Yes our standard is camelCase.  It is not documented, but we should add that.